packages/cli/src/tools/installPods.ts

```diff
@@ -103,7 +103,7 @@ async function installPods({
 }: {
   projectName: string;
   loader?: ora.Ora;
-  shouldUpdatePods: boolean;
+  shouldUpdatePods?: boolean;
```

Comment:
Updated the type definition for this property since we're checking if it is passed or not in the method execution.
I'll be happy to revert this and explicitly pass the `false` option on `commands/init/initCompat:88`.